### ISSUE TITLE
How to evaluate our model

### ISSUE THREAD

**Issue by Hiroki4869** [epic-kitchens/annotations#17]
Hello.

I'm going to try Action Recognition by using this dataset, but annotations for test set is not published(I know this is because in order to run challenge).
So, how can I test our model ?

**Comment by willprice** [comment]
Hi @Hiroki4869 ,
You can create your own validation set from the training set. Fabien Baradel et al. proposed a validation set that has been used a few times in subsequent papers (https://arxiv.org/abs/1806.06157). They use videos from P01-P25 for training and the remaining instances for validation.

**Comment by dimadamen** [comment]
Thanks Will. Just to follow up, we encourage you to find your own train/val split to tune any hyper parameters, but then train your model on the full training data.
The real test/evaluation of your model is to submit the results to the leaderboard and get the model evaluated on the test set. The test set is not published to avoid overfitting, but is open to everyone (including people not participating in the challenge) to fairly test their model's performance.

**Comment by Hiroki4869** [comment]
Thanks a lot ! That is exactly what I wanted to know.
